apps/api/src/app/events/usecases/index.ts

```diff
@@ -27,6 +28,7 @@ export const USE_CASES = [
   DigestFilterSteps,
   DigestFilterStepsRegular,
   DigestFilterStepsBackoff,
+  DigestFilterStepsTimed,
```

Comment:
the new use-case to filter steps for the timed digest

Reply:
both worker/api are using it... the api app uses it because of the batch endpoint... the worker app has the logic to handle the trigger events from the queue and parse to the jobs